VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/waiting-room-shared/waiting-room-base.component.ts

```diff
@@ -497,6 +502,27 @@ export abstract class WaitingRoomBaseDirective {
         );
     }
 
+    private setTitle(roomTransfer: RoomTransfer): void {
+        const room: string = roomTransfer.to_room;
+        const transferId = roomTransfer.participant_id;
+        if (this.participant.id === transferId) {
+            let title;
+            if (room.includes('JudgeConsultationRoom')) {
+                title = 'Video Hearings - JOH Consultation Room';
+            } else if (room.includes('JudgeJOHConsultationRoom')) {
+                title = 'Video Hearings - JOH Consultation Room';
+            } else if (room.includes('ConsultationRoom')) {
+                title = 'Video Hearings - Private Consultation Room';
+            } else if (room.includes('HearingRoom')) {
+                title = 'Video Hearings - Hearing Room';
+            } else {
+                title = 'Video Hearings - Waiting room';
```

Comment:
could we set this as the default value to reduce the number of if else conditions

Reply:
done

Reply:
done